src/file.cpp

```diff
@@ -62,6 +62,7 @@ POSSIBILITY OF SUCH DAMAGE.
 
 #ifdef TORRENT_DISK_STATS
 #include "libtorrent/io.hpp"
+#include "libtorrent/storage.hpp"
```

Comment:
right. I'm suggesting that silly_hash doesn't have to depend on whether the disk write access log is enabled or not